src/main/java/com/auth0/jwk/GuavaCachedJwkProvider.java

```diff
@@ -25,7 +25,7 @@
      * @param provider fallback provider to use when jwk is not cached
      */
     public GuavaCachedJwkProvider(final JwkProvider provider) {
-        this(provider, 5, 10, TimeUnit.HOURS);
+        this(provider, 5, 10, TimeUnit.MINUTES);
```

Comment:
`JwkProviderBuilder`'s default is still 10 hours.

https://github.com/auth0/jwks-rsa-java/blob/08e38e6e5e3c35f62c1cbcb08d21d60b4f8c2f1f/src/main/java/com/auth0/jwk/JwkProviderBuilder.java#L35